settings.gradle

```diff
@@ -61,8 +61,8 @@ dependencyResolutionManagement {
             library('amazonSTS', "software.amazon.awssdk:sts:$amazonAwsSdkVersion")
             library('amazonTranslate', "software.amazon.awssdk:translate:$amazonAwsSdkVersion")
             library('googleCloud', "com.google.cloud:libraries-bom:24.0.0")
-            library('liquibaseCore', "org.liquibase:liquibase-core:4.29.2")
-            library('liquibaseHibernate', "org.liquibase.ext:liquibase-hibernate6:4.29.2")
+            library('liquibaseCore', "org.liquibase:liquibase-core:4.26.0")
```

Comment:
Can you please revert these changes of the liquibase versions "4.29.2", but submit the revert of the "org.liquibase.gradle:org.liquibase.gradle.gradle.plugin" as the separate PR?